app/controllers/spree/admin/data_feeds_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Spree
+  module Admin
+    class DataFeedsController < ResourceController
+      def index
+        @data_feeds = Spree::DataFeed.for_store(current_store)
+                                     .accessible_by(current_ability)
+                                     .page(params[:page])
+                                     .per(params[:per_page])
```

Comment:
I'm not super familiar with the `spree_backend` codebase so this is more of a question - I see that `ResourceController` defines a `collection`:
https://github.com/spree/spree_backend/blob/a89dca15c21b38acf880bfe8355d9b932327d18e/app/controllers/spree/admin/resource_controller.rb#L192-L203

It already checks the permissions etc. Doesn't it make sense to use that here? For example:
https://github.com/spree/spree_backend/blob/a89dca15c21b38acf880bfe8355d9b932327d18e/app/controllers/spree/admin/properties_controller.rb#L1-L24
That controller adds the filtering and pagination. The one in this PR looks like a similar use case.

This begs the question - maybe it could be extracted into a mixin for example (`include Searchable/Paginable` or smth) and used across the codebase.